SQL/2016-07-06-UniquePermissionname.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE permissions ADD CONSTRAINT code_name UNIQUE (code);
```

Comment:
If an existing project already have duplicates, we could remove the them using a temporary table. And update the user_perm_rel, LorisMenuPermissions tables.